src/Cabrio.java

```diff
@@ -1,15 +1,54 @@
 public class Cabrio {
-    public static String model;
-    public static String brand;
-    public static boolean isRoofOpen;
-    public static boolean isCarMoving;
+    public String model;
+    public String brand;
+    public boolean isRoofOpen;
+    public boolean isCarMoving;
 
-    public static void toggleRoof() {
-        if (isCarMoving) {
+    public Cabrio(String model, String brand, boolean isRoofOpen, boolean isCarMoving) {
+        this.model = model;
+        this.brand = brand;
+        this.isRoofOpen = isRoofOpen;
+        this.isCarMoving = isCarMoving;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public void setModel(String model) {
+        this.model = model;
+    }
+
+    public String getBrand() {
+        return brand;
+    }
+
+    public void setBrand(String brand) {
+        this.brand = brand;
+    }
+
+    public boolean isRoofOpen() {
+        return isRoofOpen;
+    }
+
+    public void setRoofOpen(boolean roofOpen) {
+        isRoofOpen = roofOpen;
+    }
+
+    public boolean isCarMoving() {
+        return isCarMoving;
+    }
+
+    public void setCarMoving(boolean carMoving) {
+        isCarMoving = carMoving;
+    }
+
+    public static void toggleRoof(Cabrio cabrio) {
```

Comment:
Again. This method also should not be static and should not accept any params. If I have a car and i want to toggle roof i want to invoke this method on this specific car. Then why this method is static? Think before you write code.